app/models/manageiq/providers/openstack/cloud_manager/cloud_volume.rb

```diff
@@ -10,13 +10,17 @@ def self.validate_create_volume(ext_management_system)
     validate_volume(ext_management_system)
   end
 
+  def self.eligible_for_provisioning
+    super.where(:type => %w(ManageIQ::Providers::Openstack::CloudManager::CloudVolume), :bootable => true)
+  end
+
   def self.raw_create_volume(ext_management_system, options)
     cloud_tenant = options.delete(:cloud_tenant)
     volume = nil
 
     # provide display_name for Cinder V1
     options[:display_name] |= options[:name]
-    ext_management_system.with_provider_connection(cinder_connection_options(cloud_tenant)) do |service|
+    ext_management_system.with_provider_connection do |service|
       volume = service.volumes.new(options)
```

Comment:
Changing the semantics of volume creation there didn't seem related to the rest of the PR so I didn't touch it.